source/docs/software/wpilib-tools/outlineviewer/index.rst

```diff
@@ -0,0 +1,16 @@
+OutlineViewer
+=============
+
+.. image:: images/outline-viewer.png
+
+OutlineViewer is a utility used to view, modify and add to the contents of the NetworkTables for debugging purposes. It displays all key value pairs currently in the NetworkTables and can be used to modify the value of existing keys or add new keys to the table. OutlineViewer is included in the C++ and Java language updates. Teams may need to install the Java Runtime Environment to use the OutlineViewer on computers not set up for Java programming.
+
+In Visual Studio Code, press Ctrl + Shift + P and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette and select Start Tool, then select OutlineViewer.
```

Comment:
This sentence is a run-on. I would break it out a bit.

Reply:
which one?

Reply:
"In Visual Studio Code, press Ctrl + Shift + P and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette and select Start Tool, then select OutlineViewer."

Reply:
This exact same sentence is used in 5 different places if frc-docs.  I am not saying it is right but I am not sure what it should be.  I will change it for all 5.  Ideas?

Reply:
In Visual Studio Code, press Ctrl + Shift + P and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette.  Then select Start Tool, and then from the next menu select OutlineViewer.

Reply:
fixed per discord convo